examples/jsm/node-editor/inputs/TextureEditor.js

```diff
@@ -0,0 +1,145 @@
+import { LabelElement, Element, ToggleInput, SelectInput } from '../../libs/flow.module.js';
```

Comment:
It seems the import of `Element` can be removed.

Reply:
Fixed :)